tests/test_report.py

```diff
@@ -11,6 +11,32 @@ def create_report(report_type, report_start_date, report_end_date):
     )
 
 
+@pytest.mark.vcr()
+def test_report_create_with_additional_columns(report_start_date, report_end_date):
+    report = easypost.Report.create(
+        type="shipment",
+        start_date=report_start_date,
+        end_date=report_end_date,
+        additional_columns=["from_name", "from_company"],
+    )
+
+    assert isinstance(report, easypost.Report)
+    assert str.startswith(report.id, "shprep_")
```

Comment:
We don't need the ID check.
Please add a comment explaining why we're not doing any more tests (something like `Reports are queued, so we can't wait for completion. Verifying columns would require parsing CSV. Verify correct parameters via URL in cassette`)

Reply:
added